Description: Using /usr/bin/python3, not just python
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2022-09-28

Index: rabbitmq-server/deps/amqp10_common/codegen.py
===================================================================
--- rabbitmq-server.orig/deps/amqp10_common/codegen.py
+++ rabbitmq-server/deps/amqp10_common/codegen.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 from __future__ import print_function
 
Index: rabbitmq-server/deps/rabbit_common/codegen.py
===================================================================
--- rabbitmq-server.orig/deps/rabbit_common/codegen.py
+++ rabbitmq-server/deps/rabbit_common/codegen.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 ##  This Source Code Form is subject to the terms of the Mozilla Public
 ##  License, v. 2.0. If a copy of the MPL was not distributed with this
Index: rabbitmq-server/deps/rabbitmq_auth_backend_http/examples/rabbitmq_auth_backend_django/manage.py
===================================================================
--- rabbitmq-server.orig/deps/rabbitmq_auth_backend_http/examples/rabbitmq_auth_backend_django/manage.py
+++ rabbitmq-server/deps/rabbitmq_auth_backend_http/examples/rabbitmq_auth_backend_django/manage.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 import os
 import sys
 
Index: rabbitmq-server/deps/rabbitmq_auth_backend_http/examples/rabbitmq_auth_backend_django/start.sh
===================================================================
--- rabbitmq-server.orig/deps/rabbitmq_auth_backend_http/examples/rabbitmq_auth_backend_django/start.sh
+++ rabbitmq-server/deps/rabbitmq_auth_backend_http/examples/rabbitmq_auth_backend_django/start.sh
@@ -1,3 +1,3 @@
 #!/bin/sh
-python manage.py migrate
-python manage.py runserver 0.0.0.0:8000
+python3 manage.py migrate
+python3 manage.py runserver 0.0.0.0:8000
Index: rabbitmq-server/deps/rabbitmq_consistent_hash_exchange/examples/python/example1.py
===================================================================
--- rabbitmq-server.orig/deps/rabbitmq_consistent_hash_exchange/examples/python/example1.py
+++ rabbitmq-server/deps/rabbitmq_consistent_hash_exchange/examples/python/example1.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 import pika
 import time
Index: rabbitmq-server/deps/rabbitmq_consistent_hash_exchange/examples/python/example2.py
===================================================================
--- rabbitmq-server.orig/deps/rabbitmq_consistent_hash_exchange/examples/python/example2.py
+++ rabbitmq-server/deps/rabbitmq_consistent_hash_exchange/examples/python/example2.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 import pika
 import time
Index: rabbitmq-server/deps/rabbitmq_consistent_hash_exchange/examples/python/example3.py
===================================================================
--- rabbitmq-server.orig/deps/rabbitmq_consistent_hash_exchange/examples/python/example3.py
+++ rabbitmq-server/deps/rabbitmq_consistent_hash_exchange/examples/python/example3.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 import pika
 import time
